ui/nuxeo-layout.js

```diff
@@ -93,6 +93,9 @@ import './nuxeo-error.js';
       for (let i = 0; i < this.element.__templateInfo.nodeInfoList.length; i++) {
         const nodeInfo = this.element.__templateInfo.nodeInfoList[i];
         const node = this.element.__templateInfo.nodeList[i];
+        if (typeof node.hasAttribute === 'undefined') {
```

Comment:
Or we can check that it is not a function 👍

Reply:
What's the case where we have a node that doesn't have `hasAttribute`?

Reply:
If it's because we're not dealing with and element node, we should do a proper check for it instead:
```suggestion
        if (node.nodeType !== Node.ELEMENT_NODE) {
```

Reply:
I've modified the code accordingly. Tell me if there is something else you noticed and thank you a lot for the review!

Reply:
Please see https://github.com/nuxeo/nuxeo-elements/pull/194/files#r362438255